app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ProfilesController < ApplicationController
+
+    def new
+        @profile = Profile.new
+    end
+
+    def create
+        @profile = Profile.new(profile_params)
+        if @profile.save
+            redirect_to root_path
+        else
+            render :new, status: :unprocessable_entity
+        end
+    end
+
+    def show
+        puts 'profiles_id' , params[:id]
+        @profile = Profile.find(params[:id])
+    end
+
+      private
```

Comment:
Nitpick: Watch for indentation changes. `private` doesn't indicate that we're entering into a new control flow, so this feels weird to me!